recipes-devtools/python/python3-geventhttpclient_2.0.12.bb

```diff
@@ -10,6 +10,8 @@ SRC_URI = "gitsm://github.com/geventhttpclient/geventhttpclient.git;protocol=htt
            "
 SRCREV = "9c7f14c3acaec412f02cfefd067cd3802cb7ea85"
 
+S = "${UNPACKDIR}/git"
```

Comment:
I think S is no longer needed to be set if it was WORKDIR/git previously.

Reply:
I think @kraj  is right here.  Make sure you're up-to-date with latest for core layer and check this again

Reply:
I fixed all the recipes. Thanks both